src/atoms/StyledDropDown.tsx

```diff
@@ -1,6 +1,6 @@
 import { FC, useCallback, useState } from 'react'
 import { FormControl, Select, MenuItem, SelectChangeEvent } from '@mui/material'
-import StyledDropDownStyle from './StyledDropDown.style'
+import StyledDropDownStyle from './StyledDropDownStyle'
```

Comment:
My bad but I think the 
StyledDropDown.style should be named as `StyledInputBase`

Reply:
Make sure you import it as `StyledInputBase` too